sympy/physics/quantum/density.py

```diff
@@ -320,3 +276,108 @@ def fidelity(state1, state2):
 
     sqrt_state1 = state1**Rational(1, 2)
     return Tr((sqrt_state1 * state2 * sqrt_state1)**Rational(1, 2)).doit()
+    
+def bures_metric(state1, state2):
+    """ Computes the Bures metric [1]_ between two quantum states
+    The arguments provided to this function should be a square matrix or a
+    Density object. If it is a square matrix, it is assumed to be diagonalizable.
+    Parameters:
+    ==========
+    state1, state2 : a density matrix or Matrix
+    Examples:
+    =========
+    >>> from sympy.physics.quantum import TensorProduct, Ket, Dagger
+    >>> from sympy.physics.quantum.density import bures_metric
+    >>> from sympy import Matrix
+    >>> from math import sqrt
+    >>> # define qubits |0>, |1>, |00>, and |11>
+    >>> q0 = Matrix([1,0])
+    >>> q1 = Matrix([0,1])
+    >>> q00 = TensorProduct(q0,q0)
+    >>> q11 = TensorProduct(q1,q1)
+    >>> # create set of maximally entangled Bell states 
```

Comment:
It would be nice to get some whitespace in these examples, too. Perhaps you took out all the space because you were copying and pasting the examples in an interactive session? In any case, they do not present any problems in a script and are preferred for clarity of presentation.